app/src/main/java/com/github/swent/echo/viewmodels/event/EventViewModel.kt

```diff
@@ -29,30 +43,77 @@ class EventViewModel() : ViewModel() {
         )
     private val _event = MutableStateFlow<Event>(emptyEvent)
```

Comment:
Nitpicking: It would be very clean if we could use a constant like `Event.EMPTY`. We would need to add a `companion object { const val EMPTY = Event(...) }` to the `Event` data class.